common/api/core-frontend.api.md

```diff
@@ -7562,7 +7562,7 @@ export interface RealityDataSource {
     readonly realityDataType: string | undefined;
 }
 
-// @alpha (undocumented)
+// @beta
 export namespace RealityDataSource {
     // (undocumented)

```

Comment:
If you want this to be `beta`, document it.

Reply:
add documentation or mark @alpha.